sed/binning.py

```diff
@@ -25,36 +24,36 @@ def _arraysum(array_a, array_b):
 
 
 def bin_partition(
-    part: Union[dask.dataframe.core.DataFrame, pd.DataFrame],
+    part: dask.dataframe.core.DataFrame | pd.DataFrame,
     binDict: dict = None,
-    binAxes: Union[str, Sequence[str]] = None,
-    binRanges: Sequence[Tuple[float, float]] = None,
-    nBins: Union[int, Sequence[int]] = 100,
+    binAxes: str | Sequence[str] = None,
+    binRanges: Sequence[tuple[float, float]] = None,
+    nBins: int | Sequence[int] = 100,
     hist_mode: str = "numba",
     jitterParams: dict = None,
     return_edges: bool = False,
-) -> np.ndarray:
+) -> np.ndarray | tuple[np.ndarray, list]:
     """Compute the n-dimensional histogram of a single dataframe partition.
 
     Args:
-        part (ddf.DataFrame): dataframe on which to perform the histogram.
```

Comment:
why types were removed everywhere?

Reply:
As mentioned above 

> I have automated the arg types so that typehints are used to automatically include that information, as well as the default values.

There is no longer a need for including the types twice henceforth.

Reply:
This will be a little unconfortable, but I guess its ok.

I usually autogenerate the docstring, and this includes the types by default. Maybe I can find some setting that changes that.

Reply:
I thought this would be unconfortable when using autogenerated docstring strucutres, but it turns out there is a setting to use google-notypes (or whatever base style) which does the job!